src/momblish/momblish.py

```diff
@@ -45,11 +46,20 @@ def word(self, length=None):
 
         for _ in range(length-2):
             last_bigram = word[-2:]
-            next_letter = random.choices(
-                    list(self.corpus.occurences[last_bigram]),
-                    weights=self.corpus.occurences[last_bigram].values())[0]
+            try:
+                next_letter = random.choices(
+                        list(self.corpus.occurences[last_bigram]),
+                        weights=self.corpus.occurences[last_bigram].values())[0]
+            except:
+                return word
+
             word += next_letter
 
+            if word in self.blacklist:
+                word = random.choices(
```

Comment:
Feels like we could just put this in a loop with a retry rather than repeating the call?